src/components/Box/Box.spec.tsx

```diff
@@ -33,4 +33,11 @@ describe('Box', () => {
             render(<Box gridColumnGap={2} gridTemplateRows="1fr auto 10%" />).container.firstChild
         ).toMatchSnapshot();
     });
+
+    it('accepts props spreading', () => {
+        const renderBox = (props: BoxProps) => <Box {...props} />;
+        expect(
+            render(renderBox({ backgroundColor: Colors.POSITIVE_GREEN_900 })).container.firstChild
+        ).toMatchSnapshot();
```

Comment:
What is the idea behind making a snapshot? if the test checks prop spreading it should test only that the code is not crashing, no? 

Do you think we can remove the snapshot?

Reply:
No need for the snapshot, I can change the assertion to `not.toThrow()`. What do you think?